services/ui-src/src/components/fields/DropdownField.tsx

```diff
@@ -47,6 +54,29 @@ export const DropdownField = ({
     form.setValue(name, value, { shouldValidate: true });
   };
 
+  // fetch the option values and format them if necessary
+  const formatOptions = (options: DropdownOptions[] | string) => {
+    let dropdownOptions = [];
+    if (typeof options === "string") {
+      const dynamicOptionValues = report?.fieldData[options];
+      if (dynamicOptionValues) {
+        const fieldOptions = dynamicOptionValues.map((option: EntityShape) => ({
+          label: option.name,
+          value: option.id,
+        }));
+
+        dropdownOptions = fieldOptions;
+      }
+      dropdownOptions.splice(0, 0, {
+        label: dropdownDefaultOptionText,
+        value: "",
+      });
```

Comment:
Blocking: Deleting the method in forms.ts was the right call, but on lines 70-72 we're only applying the initial "Select an option" option to dynamic dropdowns. This introduces a bug of sorts, so non-dynamic dropdowns like the admin dash state selector don't have one anymore. 

<img width="548" alt="Screen Shot 2022-09-28 at 17 57 56" src="https://user-images.githubusercontent.com/27705928/192902973-bdab8e04-e6df-46ad-a15e-507e9c9b21b5.png">

Suggestion: To fix, let's either A) duplicate the code for non-string dropdown options, or B) remove the current implementation on 70-72 and instead add a conditional to check once if it's necessary for either prior to returning. I have a slight preference for option B but am also open to other ideas.